docs/design/services/dynamodb/high-level-library/README.md

```diff
@@ -0,0 +1,38 @@
+# Project Tenets (unless you know better ones)
+
+1. Meeting customers in their problem space allows them to deliver value
+   quickly.
+2. Meeting customer expectations drives usability.
+3. Discoverability drives usage.
+
+# Introduction
+
+This project provides a Dynamo DB experience that is easier to use and
+understand than can be provided by the generated Dynamo DB client. It
+provides object-oriented features, like persisting and updating Java
+data objects, as well as expressive data retrieval operations.
+
+# Project Goals
+
+1. The high-level library should be the ideal way to integrate with
+   Dynamo DB.
+2. The nouns and verbs used by the high-level library should match those
+   in Dynamo DB.
+3. The high-level library should use and build on the idiomatic norms in
+   the rest of the 2.x Java SDK.
+4. The high-level library should perform well in cold JVMs.
```

Comment:
How do we define `well`? Performing better than the low level client?

Reply:
I'm specifically trying to avoid being specific, so that we have flexibility to make the decision later on. I'm just saying that cold-start performance should be something we worry about.